CHANGELOG.md

```diff
@@ -139,7 +151,8 @@ and this project adheres to [Calendar Versioning](https://calver.org/) starting
 ## [1.0.0] - 2017-12-18
 * Fully compatible with Dgraph v1.0
 
-[Unreleased]: https://github.com/dgraph-io/dgraph4j/compare/v20.11.0...HEAD
+[Unreleased]: https://github.com/dgraph-io/dgraph4j/compare/v20.03.0...HEAD
```

Comment:
21.03.0

Reply:
Nice catch!

Reply:
The example is already in README: https://github.com/dgraph-io/dgraph4j#login-using-acl